code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -817,6 +817,52 @@
 	log_game("DYNAMIC: [key_name(S)] was spawned as a Morph by the midround ruleset.")
 	return S
 
+
+//////////////////////////////////////////////
+//                                          //
+//           PRISONER  (GHOST)            	//
+//                                          //
+//////////////////////////////////////////////
+/datum/dynamic_ruleset/midround/from_ghosts/prisoners
+	name = "Prisoners"
+	midround_ruleset_style = MIDROUND_RULESET_STYLE_LIGHT
+	role_preference = /datum/role_preference/midround_ghost/prisoner
+	required_type = /mob/dead/observer
+	enemy_roles = list(JOB_NAME_SECURITYOFFICER, JOB_NAME_DETECTIVE, JOB_NAME_WARDEN, JOB_NAME_HEADOFSECURITY, JOB_NAME_CAPTAIN)
```

Comment:
```suggestion
	enemy_roles = list(JOB_NAME_SECURITYOFFICER, JOB_NAME_WARDEN, JOB_NAME_HEADOFSECURITY)
```
Shoudln't spawn with only a captain/detective, their job isn't to actually guard the prison and I don't think there'll be much interaction.

Reply:
Faif